Widgets/4-Local Config.md

```diff
@@ -166,18 +166,23 @@ If `renderToPath: '../../../../../FORM-0/::first-child'`is applied, the widget w
 * `'white-flat'` renders a white monochrome logo with white text for medium- or dark-colored backgrounds.
 * `'purple-pill'` renders a color logo with white text on a purple background for medium- or light-colored backgrounds.
 * `'white-pill'` renders a color logo with purple text on a white background for medium- or dark-colored backgrounds.
+* `'pride-pill'` renders a color logo with purple text on a purple background for medium- or light-colored backgrounds.
+* `'pride-whitepill'` renders a color logo with purple text on a white background for medium- or dark-colored backgrounds.
```

Comment:
* `'pride-whitepill'` renders a color logo with rainbow text on a white background with rainbow border for medium- or dark-colored backgrounds.